PSReadLine.build.ps1

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Do a search and replace, remove the `All rights reserved.`

Reply:
Will do.